src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/Serialization/PropertyVisibilityTests.cs

```diff
@@ -587,6 +587,10 @@ public void PublicContextAndJsonSerializerOptions()
         [JsonSerializable(typeof(DictionaryWithPrivateKeyAndValueType))]
         [JsonSerializable(typeof(ClassWithInternalJsonIncludeProperties))]
         [JsonSerializable(typeof(ClassWithIgnoredAndPrivateMembers))]
+        [JsonSerializable(typeof(Class1))]
```

Comment:
Almost done, you also need to repeat the same declarations in the `PropertyVisibilityTestsContext_Metadata` in line 335 🙂

Reply:
Pushed a commit addressing the issue.

Reply:
@eiriktsarpalis , thank's. I've already figured it out

Reply:
Can I ask little question? Why tests of S.T.J should use JsonSerializerWrapper? Why tests fails with direct JsonSerializer usages?

Reply:
It abstracts away the underlying contract provider (reflection vs. source gen) and the underlying medium we're writing to (`string`, `byte[]`, `Stream`, async `Stream`, etc)